### ISSUE TITLE
Mixed NUnit3 & MSTest, NUnit3 not run

### ISSUE THREAD

**Issue by xhoogland** [nunit/nunit3-vs-adapter#458]
Versions:
NUnit - v3.9.0 (NuGet in testproject)
NUnit.ConsoleRunner - v3.7.0 (NuGet in testproject)
NUnit3TestAdapter - v3.9.0 (NuGet in testproject)
VS Enterprise 2017 - v15.5.7
.NET Framework - v4.6.2 (normal project as testproject)
VSTFS2013 (on-premise) - v12.0.31101.0

Currently I have 1 solution where previously all the projects had MSTest for the unit tests. Due to changing circumstances we have decided to NUnit, but not all at once. For now I have converted 1 project to NUnit since it seem to run perfectly.

Unfortunately after checking in I discovered they were not being run or recognized on our build-server, although the MSTest-tests seem to run fine. We have only MSBuild installed on our build server, so perhaps we need to install more on that server?

Kind regards,
xhoogland

**Comment by CharliePoole** [comment]
The listed software installed using nuget is sufficient. Have you added code to the build script to execute the nunit tests? How do you run them?

**Comment by rprouse** [comment]
@xhoogland I don't have TFS 2013 to test with, but you may have to add an additional test task to your TFS build definition and set the path to the NUnit adapter in the `$(Build.SourcesDirectory)\packages\` directory. Are you using the older XAML based build definitions or the newer UI/Task based ones?

**Comment by OsirisTerje** [comment]
@xhoogland  Are it working for you now?  Can we close this?

**Comment by xhoogland** [comment]
My apologies for my late- and non-reply/replies. After again, also, consulting some other people in the organization someone said they had the same problem. When we had taken a look in our buildtemplate, he saw we used some modified one.

Unfortunately some ex-colleague had decided to hardcoded/force to run MSTest...... Nonetheless to say is that we decided to use an other, standard TFS, buildtemplate we could use both. Although we "soon" migrate to our new buildserver with this product, we still wanted to have both testlibraries working since the product is quite important.

I still would like to thank you all for thinking with me. :)
